example/bpfmap/main.cpp

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
+#include <unistd.h>
+#if __linux__
+#include <linux/bpf.h>
+#include <sys/syscall.h>
+#elif defined(__APPLE__)
+#include "bpftime_epoll.h"
+#endif
+
+
+#if defined(__APPLE__)
+using namespace bpftime_epoll;
+#endif
+
+static struct bpf_insn prog[] = {
+    BPF_MOV64_IMM(BPF_REG_0, 0),
+    BPF_EXIT_INSN(),
+};
+
+static long bpf(int cmd, union bpf_attr *attr, unsigned int size) {
+    #if __linux__
+    return syscall(__NR_bpf, cmd, attr, size);
+    #endif
```

Comment:
A return value should be provided even not __linux__, or it will be a undefined behavior